### DISCUSSION TITLE
Welcome to WG Tunnel Discussions!

### DISCUSSION THREAD

**Discussion by zaneschepke** [wgtunnel/android#523] [Announcements]
## 👋 Hello!
  I'm enabling Discussions as another place for everyone to connect and share ideas.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with Zane and other community members.

  To get started, comment below.

**Comment by brunoli49** [comment]
Hi,
Nice app !
Just can't see how can I access my LAN (Nas)by smb to record with a streaming's app connected via WG Tunnel to internet.. !

**Comment by OllZ10** [comment]
I have generated a VPN tunnel from my Android device and successfully set it up. If I now activate this tunnel, an immense data transfer begins. I am in the tunnel settings, the TX value increases sharply and I see a maximum data transfer rate via LTE in the status bar. I have not started a large data transfer. Now there is no more data transfer and the tx value is at 8401.73 MB
What does that mean? Is my data transfer volume via LTE being used up?

THX for your support and for this great app
